package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "mkfifo": "^0.1.5",
+    "ndjson": "^1.5.0",
```

Comment:
I'm guessing this should be in `devDependencies` because it doesn't have a native component (I assume?) and so only needs to be installed whilst doing the webpack bundling, cc @pcarranzav